src/App.js

```diff
@@ -12,7 +12,7 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      loginDisplay: false,
+      loginDisplay: true,
```

Comment:
The reason your initial snapshot test is failing was your hardcode for skipping the login screen.